tests/integration/staking/keeper/determinstic_test.go

```diff
@@ -39,13 +39,13 @@ import (
 
 var (
 	validator1        = "cosmosvaloper1qqqryrs09ggeuqszqygqyqd2tgqmsqzewacjj7"
-	validatorAddr1, _ = sdk.ValAddressFromBech32(validator1)
+	validatorAddr1, _ = sdk.ValAddressFromHex(validator1)
```

Comment:
You might want to change the addresses from `cosmosval....` to `0x...`. 
The conversion won't work otherwise I think: https://go.dev/play/p/h-diQq1-u-b.